.github/actions/templates/validateModulePSRule/action.yml

```diff
@@ -118,14 +118,28 @@ runs:
             skipPassedRulesReport   = $false
           }
 
-          Write-Verbose "Set PS Rule Output with following parameters:`n $($Input | ConvertTo-Json -Depth 10)" -Verbose
+          Write-Verbose "Set PS Rule Output with following parameters:`n $($ParameterInput | ConvertTo-Json -Depth 10)" -Verbose
 
           # Invoke Set PSRule Output Functionality
           $null = Set-PSRuleGitHubOutput @ParameterInput
 
           Write-Output '::endgroup::'
 
-    - name: Output to GitHub job summaries
+    - name: 'Output to GitHub job summaries'
       if: always()
       shell: pwsh
-      run: Get-Content '${{ inputs.templateFilePath}}-PSRule-output.md' >> $env:GITHUB_STEP_SUMMARY
+      run: |
+        # Grouping task logs
+        Write-Output '::group::Output to GitHub job summaries'
+
+        $mdPSRuleOutputFilePath = '${{ inputs.templateFilePath}}-PSRule-output.md'
+
+        if (-not (Test-Path $mdPSRuleOutputFilePath)) {
+          Write-Warning ('Input file [{0}] not found' -f $mdPSRuleOutputFilePath)
+          return ''
+        } else {
+          Get-Content $mdPSRuleOutputFilePath >> $env:GITHUB_STEP_SUMMARY
+          Write-Verbose ('Successfully printed out file [{0}] to Job Summaries' -f $mdPSRuleOutputFilePath) -Verbose
```

Comment:
Up for debate I guess. I don't necessarily agree. I think it would be less confusing to choose one approach (since technically both work) and use the same  in all occurrences. Otherwise when updating an output to use more than one reference, or the other way around, we would also need to update the output implementation. Not ideal.

Reply:
True - neither is wrong or right. I guess I tend to try and give the code a 'somewhat as clean look as possibe' :D But as you just pointed out, that doesn't mean it's consistent. The only thing that I reaaally don't like is the 

```Powershell
"sayWhat$($iaAmSomeCodeDoSomethingWithMe | Foreach-Object { return 'no way' })kindSyntax"
```